blog/2024/keycloak-customizable-ui.adoc

```diff
@@ -0,0 +1,43 @@
+:title: Announcing New Keycloak UI Component Libraries!
+:date: 2024-09-04
+:publish: true
+:author: Erik Jan de Wit
+
+We're excited to announce the release of two new npm packages designed to supercharge your Keycloak customization efforts.
+These React component libraries, built on top of PatternFly, provide the essential building blocks for crafting account and admin consoles.
+
+The packages are:
+
+- https://www.npmjs.com/package/@keycloak/keycloak-admin-ui[@keycloak/keycloak-admin-ui]
+  - This package provides the building blocks for creating a Keycloak admin console.
+- https://www.npmjs.com/package/@keycloak/keycloak-account-ui[@keycloak/keycloak-account-ui]
+  - This package provides the building blocks for creating a Keycloak account console.
+- https://www.npmjs.com/package/@keycloak/ui-shared[@keycloak/ui-shared]
+  - This package provides shared components and utilities for building Keycloak UIs.
+
+
+== Accelerate Your Development with Our Quickstart Tool
+
+Kickstart your project with our npm create keycloak-theme my-theme command.
+This streamlined tool generates a project structure, essential dependencies, and configuration, saving you precious time.
+At the moment, the tool is only available for account consoles, but we are working on adding support for admin consoles this will be available in the next release (26.0.0).
```

Comment:
```suggestion
At the moment, the tool is only available for account consoles, but we are working on adding support for admin consoles. This will be available in the next release (26.0.0).
```